src/tools/webd/src/routes/utils.js

```diff
@@ -31,6 +31,35 @@ export const errorResponse = (res, err) => {
     .json({ error: errObj });
 };
 
+const sessionRegex = /session=([a-zA-Z0-9-]*)/;
+
+export const getSessionID = (instanceId, cookie) => {
+  instanceId = "asd"; // remove me
```

Comment:
Intentionally hardcoded?

Reply:
ah, no this was only debug code .. good catch